vcpkg.json

```diff
@@ -3,7 +3,17 @@
   "version-string": "0.1",
   "description": "Cisco MLS C++ library",
   "dependencies": [
-    "openssl",
+    {
+      "name": "openssl",
+      "version>=": "1.1.1n"
+    },
     "doctest"
+  ],
+  "builtin-baseline": "3b3bd424827a1f7f4813216f6b32b6c61e386b2e",
```

Comment:
What is this hash?

Reply:
this is vcpkg baseline commit that included support for openssl1.1h

Reply:
https://vcpkg.io/en/docs/examples/versioning.getting-started.html